packages/gatsby-theme-newrelic/src/components/Collapser.js

```diff
@@ -1,16 +1,23 @@
-import React, { useState, useMemo } from 'react';
+import React, { useState, useMemo, useEffect } from 'react';
```

Comment:
```suggestion
import React, { useState, useMemo } from 'react';
```

Reply:
There also seems to be some tests failing 🤔

Reply:
ah yeah i see that. looks like the jest test for Collapsers is erroring due to using the hook `useQueryParams` which uses `useLocation`. so maybe it sees that the Collapser has some code in it that looks at / relies on location, but test doesn't handle that? any idea how to address that in a jest test?

Reply:
i added `<LocationProvider>` to the collapser test components and it doesn't fail now. not exactly sure if this is the optimal solution though?